process/transaction/baseProcess.go

```diff
@@ -150,6 +151,10 @@ func (txProc *baseTxProcessor) checkTxValues(
 		txFee = txProc.economicsFee.ComputeTxFee(tx)
 	}
 
+	if !txProc.shouldCheckBalanceHandler() {
```

Comment:
it is not advisable to have this kind of skips.

Reply:
removed.